sdks/java/core/src/test/java/org/apache/beam/sdk/util/SerializableUtilsTest.java

```diff
@@ -51,6 +60,52 @@ public SerializableByJava(String stringValue, int intValue) {
   }
 
   @Test
+  public void customClassLoader() throws Exception {
```

Comment:
This is a much cleaner way to test custom class loaders: https://github.com/apache/beam/blob/b2138b0d751967522814fcd4ccdc8a3bd2945c5a/sdks/java/core/src/test/java/org/apache/beam/sdk/coders/AvroCoderTest.java#L200

Reply:
Sorry I dont get this one since it does ~the same with just test adaptations

Reply:
I was suggesting to copy the pattern and make InterceptingUrlClassLoader a public class which can be used as a testing utility for testing multiple class loader scenarios.

Reply:
Done, thanks for explaining